gradle/libs.versions.toml

```diff
@@ -31,8 +31,9 @@ managed-jdbi = "3.45.4"
 
 # JDBC Drivers
 
-managed-ojdbc = "21.15.0.0"
+managed-ojdbc = "23.5.0.24.07"
 managed-ojdbcdms = "21.15.0.0"
+managed-dms = "23.3.0.23.09"
```

Comment:
`com.oracle.database.observability:dms` dependency was using the same version as other ojdbc dependencies but now dms has got latest `23.3.0.23.09` and doesn't have `23.5.0.24.07` so I think it makes sense to define separate dependency now.

Reply:
Thanks @radovanradic It makes sense to define it separately.